Pinta/Actions/View/EditCanvasGridAction.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Pinta.Core;
+using Pinta.Core.Managers;
+
+namespace Pinta.Actions;
+
+internal sealed class EditCanvasGridAction : IActionHandler
+{
+	private readonly ViewActions view;
+	private readonly ChromeManager chrome;
+	private readonly CanvasGridManager canvas_grid;
+
+	internal EditCanvasGridAction (
+		ViewActions view,
+		ChromeManager chrome,
+		CanvasGridManager canvasGrid)
+	{
+		this.view = view;
+		this.chrome = chrome;
+		canvas_grid = canvasGrid;
```

Comment:
Minor nitpick - the style is inconsistent here with using a mix of `this.foo = ...` and just `foo = ...`